### PR TITLE
qa/vstart_runner: minor bug fix and minor adjustments

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
The message regarding deletion of helper tools is printed for every
command. This message should be printed only when applicable.
    
Besides -
    
* Move XXX comments to _do_run() since it increases visibility of
  these messages.
* Move omission of arguments stuff to new method clear up the clutter.
* And remove shell as a parameter from _perform_checks_and_adjustments
  since it's redundant.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/tasks/vstart_runner.py [hunk 1]
```diff
@@ -394,41 +394,42 @@ def sudo_write_file(self, path, data, **kwargs):
...
+            if i in args:
+                helper_tools_found = True
+                break
+        else:
+            helper_tools_found = False
 
-        # we're intentionally logging the command before overriding to avoid
-        # this clutter in the log.
-        log.debug('> ' + args)
+        if not helper_tools_found and not 'sudo' in args:
```
Comment (dparmar18) [inline qa/tasks/vstart_runner.py:412]: do you mean `if not helper_tools_found and 'sudo' not in args`?

Comment (dparmar18) [reply qa/tasks/vstart_runner.py:412]: Rest LGTM

Comment (rishabh-d-dave) [reply qa/tasks/vstart_runner.py]: Yep, fixed.

**Comment by rishabh-d-dave** [comment]
https://gist.github.com/rishabh-d-dave/c51223bef9b6277f404a52a842523534

**Comment by rishabh-d-dave** [comment]
https://gist.github.com/rishabh-d-dave/c4731bda4d8873ef5a1b6b3971368ad2

**Review by vshankar — APPROVED** [review]

**Pull request merged**